constants/auth.go

```diff
@@ -0,0 +1,19 @@
+package constants
+
+// OTPExpiryTime is OTP expiry time in seconds...
+const OTPExpiryTime = 300
+
+// OTPReCreationTime ...
+const OTPReCreationTime = 120
+
+// EmailUsername ...
+const EmailUsername = "sandipagarwal"
```

Comment:
Please move them (username and password) to environment variables.

Reply:
done.